app/javascript/packages/document-capture/components/in-person-location-step.jsx

```diff
@@ -0,0 +1,26 @@
+import { PageHeading } from '@18f/identity-components';
```

Comment:
Can we implement this (and all other components) as `.tsx` ? It's fine to mix `.tsx` into existing packages, even if other files in it are `.jsx`.

Reply:
addressed in 027b25e3f8eefea84e70645f48a53f19d5a21aba